includes/definitions.inc.php

```diff
@@ -240,6 +240,18 @@
 $config['os'][$os]['over'][2]['graph'] = 'device_mempool';
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
+$os = 'cumulus';
+$config['os'][$os]['type']             = 'network';
+$config['os'][$os]['group']            = 'unix';
+$config['os'][$os]['text']             = 'Cumulus Linux';
+//$config['os'][$os]['icon']             = 'cumulus';
```

Comment:
Well, I am waiting for an official green light to push the icon from Cumulus Network.
Do I remove until I get it ? Please advise.